src/App.tsx

```diff
@@ -60,15 +67,36 @@ function App(): ReactElement {
   const [colorRampMin, setColorRampMin] = useState(0);
   const [colorRampMax, setColorRampMax] = useState(0);
   const [outOfRangeDrawSettings, setoutOfRangeDrawSettings] = useState({
-    mode: DrawMode.USE_RAMP,
+    mode: DrawMode.USE_COLOR,
     color: new Color(OUT_OF_RANGE_COLOR_DEFAULT),
   });
   const [outlierDrawSettings, setOutlierDrawSettings] = useState({
     mode: DrawMode.USE_COLOR,
     color: new Color(OUTLIER_COLOR_DEFAULT),
   });
-  const [playbackFps, setPlaybackFps] = useState(DEFAULT_PLAYBACK_FPS);
 
+  const [featureThresholds, _setFeatureThresholds] = useState<FeatureThreshold[]>([]);
+  const setFeatureThresholds = useCallback(
+    // Change the current feature min + max on the color ramp if that feature's threshold moved.
+    (newThresholds: FeatureThreshold[]): void => {
+      // Check if the current feature name is being thresholded on, and if that threshold
+      // has changed. If so, snap the current min + max color ramp values so they match the new
+      // threshold values.
+      const currThreshold = featureThresholds.find((t) => t.featureName === featureName);
+      const newThreshold = newThresholds.find((t) => t.featureName === featureName);
```

Comment:
Ah, so that's intended behavior. The thought there is that the color map range should by default show the range of what's being thresholded.

I updated the `updateFeature` method so it now sets the main range slider to match the threshold on initial load. Let me know if this makes sense?